code/modules/antagonists/voidwalker/voidwalker_abilities.dm

```diff
@@ -0,0 +1,194 @@
+/// Remain in someones view without breaking line of sight
+/datum/action/cooldown/spell/pointed/unsettle
+	name = "Unsettle"
+	desc = "Stare directly into someone who doesn't see you. Remain in their view for a bit to stun them for 2 seconds and announce your presence to them. "
+	button_icon_state = "terrify"
+	background_icon_state = "bg_alien"
+	overlay_icon_state = "bg_alien_border"
+	panel = null
+	spell_requirements = NONE
+	cooldown_time = 8 SECONDS
+	cast_range = 9
+	active_msg = "You prepare to stare down a target..."
+	deactive_msg = "You refocus your eyes..."
+	/// how long we need to stare at someone to unsettle them (woooooh)
+	var/stare_time = 8 SECONDS
+	/// how long we stun someone on succesful cast
+	var/stun_time = 2 SECONDS
+	/// stamina damage we doooo
+	var/stamina_damage = 80
+
+/datum/action/cooldown/spell/pointed/unsettle/is_valid_target(atom/cast_on)
+	. = ..()
+
+	if(!ishuman(cast_on))
```

Comment:
How come? Any reason you _shouldn't_ be able to terrify Ian? Looking below, doesn't seem like anything you are checking is human-only.
Maybe make it check for mechanical/spectral biotypes and allow everything else, if possible?